e2e-tests/test/hardhat-apis.test.ts

```diff
@@ -64,6 +64,24 @@ describe("hardhat_mine", function () {
   });
 });
 
+describe("hardhat_setNextBlockBaseFeePerGas", function () {
+  it("should allow setting the next block's base fee per gas", async function () {
+    // Arrange
+    const price = "0x12d687";
```

Comment:
should we play with types like here? https://github.com/NomicFoundation/hardhat/blob/599a285b37a27cd4a99fd33ef9f4cdad92466af7/packages/hardhat-network-helpers/test/helpers/setNextBlockBaseFeePerGas.ts#L29

Reply:
I'd say no; when testing the server, it doesn't really matter what the types on the client are - the important thing are the communicated JSON strings... Of course, that might be just my personal JS preference...